src/Decompiler/Analysis/IndirectCallRewriter.cs

```diff
@@ -148,6 +149,28 @@ private void RewriteCall(Statement stm, CallInstruction call)
             Use(adjustRegStm, src);
         }
 
+        private void DefineTrashedIdentifiers(
```

Comment:
Consider renaming this to `DefineUninitializedIdentifiers`

Reply:
Done